OpenCloudSDK/Core/Sharing/OCCore+Sharing.m

```diff
@@ -388,8 +388,8 @@ - (nullable NSProgress *)makeDecisionOnShare:(OCShare *)share accept:(BOOL)accep
 					 customizablePermissions:OCSharePermissionsMaskNone
 						       locations:OCLocationTypeFile|OCLocationTypeFolder
 						      symbolName:@"eye.fill"
-						   localizedName:OCLocalizedString(@"Viewer",nil)
-					    localizedDescription:OCLocalizedString(resharingSupported ? @"Download, preview and share" : @"Download and preview", nil)],
+						   localizedName:OCLocalizedString(@"Can view",nil)
+					    localizedDescription:OCLocalizedString(resharingSupported ? @"Download, preview and share." : @"View and download.", nil)],
```

Comment:
We don't support resharing anymore, but to keep the code changes as simple as possible, we keep the original strings and just modify the regular sharing role descriptions